src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/no.authorization.sends.cache.control.private.except.when.public/accept/client.rpt

```diff
@@ -79,6 +79,7 @@ write nukleus:begin.ext ${http:beginEx()
                               .header("cache-control", "no-cache")
                               .header("prefer", "wait=100")
                               .header("if-none-match", etag)
+                              .header("authorization", "some-authorization-value")
```

Comment:
These are the mismatches, they were passing because there were typos in the tests instead of `accept` it was using `connect`.